src/main/java/appeng/container/slot/SlotFake.java

```diff
@@ -27,7 +27,7 @@
 public class SlotFake extends AppEngSlot
 {
 
-	final int invSlot;
+	private final int invSlot;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "invSlot" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)